guides/source/getting_started_with_devcontainer.md

```diff
@@ -0,0 +1,123 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
```

Comment:
@andrewn617 it just occurred to me that we need to [wrap the file to 80 columns](https://edgeguides.rubyonrails.org/ruby_on_rails_guides_guidelines.html#column-wrapping) if you have not done that already. 

Please exclude line 1 from this wrapping. Thanks!